kfdefs/overlays/moc/smaug/opf-jupyterhub-stage/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: opf-jupyterhub-stage
+
+resources:
+  - ../../../../base/jupyterhub
+
+patchesJson6902:
+  - patch: |
+      - op: add
+        path: /spec/applications/2/kustomizeConfig/overlays/-
+        value: storage-class
+      - op: add
+        path: /spec/applications/2/kustomizeConfig/parameters/-
+        value:
+          name: storage_class
+          value: ocs-storagecluster-cephfs
```

Comment:
Waiting on pvcs being setup on Smaug, need to replace line 18

Reply:
@Gregory-Pereira you can set the `value` to `moc-nfs-csi`